docs/python_api/py_acados_classes.dot

```diff
@@ -0,0 +1,47 @@
+// This file is used to draw the dependency graph of acados classes in the Python API documentation
+
+// the graph
+digraph ocp {
+        // some graph-specific definitions
+        graph [
+               rankdir = "LR"
+               ];
+
+        // define solver nodes
+        solver  [label="AcadosOcpSolver"
+                 href="https://docs.acados.org/python_api/index.html#acados_template.acados_ocp_solver.AcadosOcpSolver"];
```

Comment:
I really like this Overview graphic!
But I think the link should work differently, i.e. I would want the browser to jump down to the documentation of these classes when clicking on them.
Instead the documentation of the class is opened where the graphic is.
Note, I already pushed the documentation from your branch, so you can check:
https://docs.acados.org/python_api/index.html#acados_template.acados_ocp_solver.AcadosOcpSolver
If you can suggest a fix for this, it would be highly appreciated!

Reply:
I see what you mean. That is weird, it was working before the update on the website.
I have an idea what it could be (at least it fixed it for me):
```suggestion
                 href="https://docs.acados.org/python_api/index.html#acados_template.acados_ocp_solver.AcadosOcpSolver", target="_top"];
```
You have to add `, target="_top"` after each url below as well.

Reply:
great, that worked.
https://github.com/FreyJo/acados/pull/66/commits/0a0468d67031f6a609d24ab0b1e7660a3428a39f

I will merge this into acados/master soon.